pkgs/by-name/le/leftwm-theme/package.nix

```
@@ -0,0 +1,38 @@
+{ stdenv, lib, fetchFromGitHub, rustPlatform, openssl, pkg-config }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "leftwm-theme";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "leftwm";
+    repo = "leftwm-theme";
+    rev = "refs/tags/${version}";
```

Missed the `v` in rev (my bad on previous suggestion)

```suggestion
    rev = "refs/tags/v${version}";
```

---

Aand also on it!